skyportal/handlers/api/internal/profile.py

```diff
@@ -27,4 +29,41 @@ def get(self):
         return self.success(data={'username': self.current_user.username,
                                   'roles': user_roles,
                                   'acls': user_acls,
-                                  'tokens': user_tokens})
+                                  'tokens': user_tokens,
+                                  'preferences': self.current_user.preferences})
+
+    def put(self):
```

Comment:
We should probably define a schema for the configuration, so that these can be validated.